rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
```

Comment:
- [ ] It's required that the rental adds itself to the book and the person respectively. Kindly implement that in your initialize method. 

`book.rentals << self`
`person.rentals << self`

Let's add those in the initialize method 👍🏽